hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -144,6 +144,7 @@ def get_metrics(self) -> nn.ModuleDict:
                                   MetricsKey.ACC_MACRO: Accuracy(num_classes=self.n_classes, average='macro'),
                                   MetricsKey.ACC_WEIGHTED: Accuracy(num_classes=self.n_classes, average='weighted'),
                                   MetricsKey.AUROC: AUROC(num_classes=self.n_classes),
+                                  MetricsKey.COHENKAPPA: CohenKappa(num_classes=self.n_classes, weights='quadratic'),
```

Comment:
It's not immediately obvious that this corresponds to the same quadratic-weighted kappa used in the PANDA benchmark. Because Cohen's kappa is most often used in binary classification (cf. [Wikipedia](https://en.wikipedia.org/wiki/Cohen%27s_kappa)), this can cause some confusion (it did for me!).

Consider adding some documentation (as a comment here and/or in a readme) maybe with a reference link explaining how they're related. A complementary option is to rename the key to `QUAD_KAPPA` or whatever for clarity, if you think it makes sense.

Reply:
I think `COHEN_KAPPA` is more general and makes more sense. I added the comment with a link of the documentation which explains specifically QWK. I also added this comment in the PR description.